.dockerignore

```diff
@@ -29,3 +29,7 @@ db.sqlite3
 
 # Font-end
 **/node_modules
+
+# Assets
+src/ashley/static/ashley/js
+src/ashley/static/ashley/css
```

Comment:
Maybe you want to add the `fonts` directory, since it's also generated by the frontend ?

Reply:
true, thank you